host_vars/mlk-nk-nf-wbp1/base.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+location: mlk_nk
```

Comment:
Please decide on wether you want to use `_` oder `-` here. I suppose you wanted to have a dash.